source/_components/switch.tesla.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: "Tesla Switch"
+description: "Instructions on how to integrate Tesla charging switch into Home Assistant."
+date: 2017-09-19 12:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Switch
+ha_iot_class: "Cloud Push"
+ha_release: 0.54
+---
+
+The `Tesla` platform allows you to control your [Tesla](https://www.tesla.com/) charger (enable/disable charging) from within Home Assistant.
+
+The lock platform will be automatically configured if Tesla component is configured.
```

Comment:
Copy/paste error error. Needs to be "switch platform"?

Reply:
Thank you. Fixed.